evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -216,3 +216,10 @@ def has_nonresponsible_editor(evaluation):
 @register.filter
 def order_by(iterable, attribute):
     return sorted(iterable, key=lambda item: getattr(item, attribute))
+
+
+@register.filter
+def value_at(dictionary, key):
+    if key in dictionary:
+        return dictionary[key]
+    return None
```

Comment:
We still wanted to use `get` for name and implementation (https://github.com/e-valuation/EvaP/pull/1718#discussion_r817201881) :)